packages/pipeline-editor/src/CustomFormControls/StringArrayControl.tsx

```diff
@@ -388,6 +399,16 @@ function StringArrayControl({ placeholder, format }: Props) {
               Browse
             </button>
           )}
+          {format === "properties" && (
```

Comment:
yea, let's change this to a `canXXX` boolean like mentioned above. The `format` prop is more intended for potential regex of common formats like date, time, email, etc...

Reply:
done, will be in next commit